online-dashboard-api/app/Eums/BookingStatus.php

```diff
@@ -25,4 +25,26 @@ public static function getDescription($value): string
         };
 
     }
+
+    public static function getAll()
+    {
```

Comment:
Heavy work 😢

Remove all the getAll() and add a function based on the class in the baseenum file.